src/extensions/hive/components/database/Dropzone.css

```diff
@@ -0,0 +1,34 @@
+
+.dropzone {
```

Comment:
Also here I'd prefer to just not have .css files in the `src/` directory

Reply:
can move to inline styling